tests/integration_test/integration_test.sh

```diff
@@ -398,36 +391,42 @@ ss2_workflow_id=$(docker run --rm -v $script_dir:/app \
 
 printf "\nss2_workflow_id: $ss2_workflow_id"
 
-# 10. Poll for completion
+# 9. Poll for completion
 printf "\n\nAwaiting workflow completion\n"
 
 if [ $use_caas ]; then
```

Comment:
I think this needs to be `if [ "$use_caas" == "true" ]`. In current form it seems to always evaluate to true.

Reply:
The default for `$use_caas` is an empty string which evaluates to false, but I agree it's better to be explicit so that "false" does not evaluate to true here. 

To fix the Lira config template, I could set the default for `use_caas` to `false` instead of an empty string, and remove the quotes.